packages/appium/docs/en/cli/index.md

```diff
@@ -2,4 +2,12 @@
 title: CLI Intro
 ---
 
```

Comment:
actually, `appium` is just an executable shell script, which is created in the default local bin ([usually](https://dev.to/kcdchennai/demystifying-linux-the-purpose-of-bin-usrbin-and-usrlocalbin-5a8e) `/usr/local/bin`, ).

Reply:
good point, i made a more nuanced statement.